content/app/testing/local/_index.en.md

```diff
@@ -5,26 +5,21 @@ description: Description of how test of an app locally on your own machine.
 weight: 100
 aliases:
 - /app/testing/local/
-tags: [translate-to-english]
 ---
 
-Dersom man skal skrive en del kode (f.eks. [logikk](/app/development/logic/)), eller kjapt sjekke hvordan skjema ser ut kan det være nytting å kunne teste endringer uten å måtte deploye hele appen til testmiljø.
+If you need to write a significant amount of code (e.g., [logic](/app/development/logic/)), or quickly check how a form looks, it can be useful to test changes without having to deploy the entire app to a test environment.
```

Comment:
```suggestion
If you need to write a significant amount of code (e.g., [logic](/app/development/logic/)), or quickly check how a form looks, it can be useful to test changes without having to deploy the app to a test environment.
```